test/manual/mongodb_oidc.test.ts

```diff
@@ -0,0 +1,456 @@
+import { readFile } from 'node:fs/promises';
+
+import { expect } from 'chai';
+
+import { MongoClient, MongoInvalidArgumentError, OIDC_WORKFLOWS } from '../mongodb';
+
+describe('MONGODB-OIDC', function () {
+  context('when running in the environment', function () {
+    it('contains AWS_WEB_IDENTITY_TOKEN_FILE', function () {
+      expect(process.env).to.have.property('AWS_WEB_IDENTITY_TOKEN_FILE');
+    });
+  });
+
+  describe('OIDC Auth Spec Prose Tests', function () {
```

Comment:
since these are the prose tests, can we name the file in accordance with the test convention? i.e., `mongodb_oidc.prose.test`?

Reply:
Renamed.